servers/http/package.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "node-example-http",
+  "version": "1.0.0",
+  "description": "A simple http server with Node.js",
+  "keywords": [],
+  "author": "Patrick Heneise <patrick@zentered.co>",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "test": "node --test"
+  }
```

```suggestion
  },
  "engines": {
    "node": "20"
  }
```
I'm not sure about this code but we need this